index.html

```diff
@@ -1367,14 +1367,44 @@ <h2>Help your friends find some good learning resources</h2>
                 </div>
             </div>
             <!-- ________Guilherme Orcezi Contributor card END ________  -->
+                 <!--contributor card end-->
 
-                <!--contributor card end-->
+          <!-- ________Keegan Colaco Contributor card START ________  -->
+            <div class="card">
+                <p class="name">Keegan Colaco</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/KeeganColaco20" target="_blank">@KeeganColaco20</a>
+                </p>
+                <p class="about">
+                    I'm Keegan Colaco a Computer Engineering student   having keen interest in Frontend Web Development. Always Up for working with cool open source Projects.
```

Comment:
can you remove the white space in your text

Reply:
Yh i will remove them